#!/bin/bash

PATCH="plymouth-usr-merge.patch"
for i in sys-boot/plymouth/plymouth-*.ebuild; do
	cat > ${PATCH} << EOF
--- a/${i}
+++ b/${i}
@@ -73,9 +73,4 @@ src_install() {
 	insinto /usr/share/plymouth
 	newins "\${DISTDIR}"/gentoo-logo.png bizcom.png
 
-	# Install compatibility symlinks as some rdeps hardcode the paths
-	dosym /usr/bin/plymouth /bin/plymouth
-	dosym /usr/sbin/plymouth-set-default-theme /sbin/plymouth-set-default-theme
-	dosym /usr/sbin/plymouthd /sbin/plymouthd
-
 	readme.gentoo_create_doc
EOF
	git apply ${PATCH} || exit 1
	rm ${PATCH}
done
